### PR TITLE
ENG-2460: Add ProtocolType to ProtocolComponent Protobuf msg

### PR DESCRIPTION
**Pull request by SmartMonkey-git**
[Task](https://datarevenue.atlassian.net/browse/ENG-2460)

### CODE REVIEW

File: tycho-indexer/src/pb/tycho.evm.v1.rs [hunk 1]
```diff
@@ -104,62 +212,82 @@ pub struct TransactionEntityChanges {
...
 #[derive(Clone, PartialEq, ::prost::Message)]
 pub struct BlockEntityChanges {
+    /// The block for which these changes are collectively computed.
     #[prost(message, optional, tag="1")]
     pub block: ::core::option::Option<Block>,
+    /// The set of transaction changes observed in the specified block.
     #[prost(message, repeated, tag="2")]
     pub changes: ::prost::alloc::vec::Vec<TransactionEntityChanges>,
 }
+// This file contains proto definitions specific to the VM integration.
```
Comment (tamaralipows) [inline tycho-indexer/src/pb/tycho.evm.v1.rs:233]: Why is this here not at the top of the file if saying something about the file?

Comment (SmartMonkey-git) [reply tycho-indexer/src/pb/tycho.evm.v1.rs:233]: They are auto generated. 🤔

Comment (SmartMonkey-git) [reply tycho-indexer/src/pb/tycho.evm.v1.rs:233]: I just gonna remove them, but it would be awful if they get generated every time we adjust the buf messages :no_good:

Comment (tamaralipows) [reply tycho-indexer/src/pb/tycho.evm.v1.rs:233]: Oh damn no you are right, ignore this
